src/App.jsx

```diff
@@ -1,15 +1,26 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import { Routes, Route } from 'react-router-dom'
+import Header from './components/Header'
+import Navbar from './components/Navbar'
+import Home from './components/Home'
+import Articles from './components/Articles'
+import ArticleShowCard from './components/ArticleShowCard'
 import './App.css'
 
 function App() {
   const [count, setCount] = useState(0)
 
   return (
-    <>
-    
-    </>
+    <div>
+      <Header />
+      <Navbar />
+      <Routes>
+        <Route path = "/" element = {<Home />} />
+        <Route path = "/articles" element = {<Articles />} />
+        <Route path = "/article" element = {<ArticleShowCard />} />
```

Comment:
This route is a bit confusing. Should it be a single article? In that case, it will need an article id and this is a future ticket 😁 Please get rid of this route for now